jupyter_client/connect.py

```diff
@@ -158,6 +155,8 @@ def find_connection_file(filename='kernel-*.json', path=None):
     -------
     str : The absolute path of the connection file.
     """
+    if profile is not None:
+        warnings.warn("Jupyter has no profiles. profile=%s has been ignored." % profile)
```

Comment:
I'd probably use `%r`, so that it shows up as `profile='foo'` rather than `profile=foo`.